src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -53,7 +53,7 @@
     ImportInterface, ContainerSymbol
 from psyclone.psyir.tools import DependencyTools
 from psyclone.psyir.transformations import ProfileTrans, RegionTrans, \
-    TransformationError
+    TransformationError, ParallelLoopTrans
```

Comment:
pylint says ParallelLoopTrans is not used

Reply:
Fixed.